src/main/webapp/i18n/de/iris.json

```
@@ -128,6 +129,9 @@
                 "pyrisUnavailable": "Iris ist derzeit nicht verfügbar.",
                 "pyrisError": "Ein Fehler ist bei der Kommunikation mit Iris aufgetreten.",
                 "transcriptionIngestionError": "Fehler beim Senden einer Vorlesungstranskription"
+            },
+            "controlCenter": {
+                "tooltip": "Durch das Umschalten aktivierst oder deaktivierst du ALLE Iris Features für diesen Kurs. Für eine granularere Kontrolle nutze bitte die Iris Einstellungen, indem du auf Benutzerdefiniert klickst."
```

Maybe this sounds a little smoother and less technical? 
```suggestion
                "tooltip": "Durch das Umschalten aktivierst oder deaktivierst du ALLE Iris Features für diesen Kurs. Für eine gezieltere Kontrolle nutze bitte die Iris Einstellungen, indem du auf Benutzerdefiniert klickst."
```

---

Not sure if it is good to hardcode `Benutzerdefiniert` as it probably wont be updated when `"custom": "Benutzerdefiniert",` changes.

Can we pass it as parameter instead?

---

Also "click button yz" reads cumbersome, I would rather say "open the custom settings"

---

adapted the translation. Passing it as value is not feasible as it's not the standard translate directive but an input for the help-icon compoonent.